src/app/pcasts/controllers/facebook_sign_in_controller.py

```diff
@@ -10,7 +10,7 @@ def get_methods(self):
     return ['POST']
 
   def content(self, **kwargs):
-    token = request.args['access_token']
+    token = request.data
```

Comment:
Isn't data the entire body?

Reply:
Yes data is the entire body. We could also make it something structured which can help with error handling, make things simpler later on if we want to add something to the body.

Reply:
imo it should be more structured, just in case in the future we want to add some other info like token expiration date or refresh token or smth